README.md

```diff
@@ -766,6 +767,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 ## Polls and Events
 
+  * Calagator - An event aggregator. ([Demo](http://calagator.org/), [Source code](https://github.com/calagator/calagator)) `MIT` `Ruby`
```

Comment:
Please follow the guidelines in [Contributing](https://github.com/Kickball/awesome-selfhosted/blob/master/.github/CONTRIBUTING.md) for example:
> \[Calagator\]\(http://calagator.org/)